src/lib/es2020.intl.d.ts

```diff
@@ -1,456 +1,240 @@
 /// <reference lib="es2018.intl" />
 declare namespace Intl {
-    /**
-     * A string that is a valid [Unicode BCP 47 Locale Identifier](https://unicode.org/reports/tr35/#Unicode_locale_identifier).
-     *
-     * For example: "fa", "es-MX", "zh-Hant-TW".
-     *
-     * See [MDN - Intl - locales argument](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#locales_argument).
-     */
-    type UnicodeBCP47LocaleIdentifier = string;

```

Comment:
todo: either restore or add deprecated type export.

While this type doesn't do any harm, primitive aliasing feels like a paradigm that should either be used everywhere in the library, or not used at all. This is the only case of it throughout lib.d.ts.